Makefile

```diff
@@ -29,7 +29,7 @@ test-install: clean
 	$(PIP) install --requirement $(TEST_REQUIREMENTS)
 
 test:
-	pytest
+	dbus-launch pytest
```

Comment:
dbus-launch doesn't come preinstalled with OSX
You need to manually install it with thirdparty package managers like `homebrew`